contracts/test/Gateway.t.sol

```diff
@@ -893,7 +912,7 @@ contract GatewayTest is Test {
         assertEq(fee, 20000000000000001);
     }
 
-    function testSendTokenToForeignDestWithInvalidFee() public {
+    function testSendTokenWithZeroFeeToForeignDestWithInvalidFee() public {
```

Comment:
This naming is a bit awkward, I'd rather do `testSendTokenWithZeroInvalidFeeToForeignDest`. But this is bikeshedding 🤷🏻‍♀️

Reply:
Addressed in [23fe816](https://github.com/Snowfork/snowbridge/pull/1168/commits/23fe816ce2d5ac3e1242e87a27f816571dcee77e)